src/azure-cli-telemetry/azure/cli/telemetry/tests/resources/cache.2

```diff
@@ -0,0 +1 @@
+2018-06-05T16:40:29,{"c4395b75-49cc-422c-bc95-c7d51aef5d46":[{"name":"azurecli/command","properties":{"Reserved.DataModel.EntityType":"UserTask","Reserved.DataModel.Action.Type":"Atomic","Reserved.DataModel.Action.Result":"Success","Reserved.ChannelUsed":"AI","Reserved.EventId":"e4386935-634c-41fc-9bb1-2ab08435e65e","Reserved.SequenceNumber":1,"Reserved.SessionId":"57ee337b-ed92-4601-9253-37a1a08d304a","Reserved.TimeSinceSessionStart":0,"Reserved.DataModel.Source":"DataModelAPI","Reserved.DataModel.EntitySchemaVersion":4,"Reserved.DataModel.Severity":0,"Reserved.DataModel.ProductName":"azurecli","Reserved.DataModel.FeatureName":"command","Reserved.DataModel.EntityName":"cloud-list","Reserved.DataModel.CorrelationId":"aa7d4208-122a-48f6-9915-28e275b5121c","Context.Default.VS.Core.ExeName":"azurecli","Context.Default.VS.Core.ExeVersion":"2.0.35@none","Context.Default.VS.Core.MacAddressHash":"96728d3db37e5906434442d655caa4ce6d95a8ebfbabc682ef8afd5860f4774f","Context.Default.VS.Core.Machine.Id":"96728d3d-b37e-5906-4344-42d655caa4ce","Context.Default.VS.Core.OS.Type":"darwin","Context.Default.VS.Core.OS.Version":"darwin kernel version 17.5.0: fri apr 13 19:32:32 pdt 2018; root:xnu-4570.51.2~1/release_x86_64","Context.Default.VS.Core.User.Id":"4174e03a-3119-11e8-9f09-4c32759c53db","Context.Default.VS.Core.User.IsMicrosoftInternal":"False","Context.Default.VS.Core.User.IsOptedIn":"True","Context.Default.VS.Core.TelemetryApi.ProductVersion":"azurecli@2.0.35","Context.Default.AzureCLI.Source":"az","Context.Default.AzureCLI.ClientRequestId":"d3e57578-6919-11e8-86ec-4c32759c53db","Context.Default.AzureCLI.CoreVersion":"2.0.35","Context.Default.AzureCLI.InstallationId":"4174e03a-3119-11e8-9f09-4c32759c53db","Context.Default.AzureCLI.ShellType":"/bin/zsh","Context.Default.AzureCLI.UserAzureId":"9451d88fb6ed17c3a8b03c61e98b4183cee8d32ce04039f4adb9887fbfaf9be2","Context.Default.AzureCLI.UserAzureSubscriptionId":"6b085460-5f21-477e-ba44-1035046e9101","Context.Default.AzureCLI.DefaultOutputType":"table","Context.Default.AzureCLI.EnvironmentVariables":"[]","Context.Default.AzureCLI.Locale":"en_US,UTF-8","Context.Default.AzureCLI.StartTime":"2018-06-05 23:40:28.477505","Context.Default.AzureCLI.EndTime":"2018-06-05 23:40:29.365485","Context.Default.AzureCLI.OutputType":"table","Context.Default.AzureCLI.RawCommand":"cloud list","Context.Default.AzureCLI.Params":"","Context.Default.AzureCLI.PythonVersion":"3.6.4"}}]}
```

Comment:
Maybe random, but what scenario made the the second cache contain less telemetry than `cache.1` as well as the older caches?

Reply:
no sure. but it is not relevant to the upload process. it doesn't concern the file size.